src/components/Cart/CartSummary.jsx

```diff
@@ -1,52 +1,52 @@
-import React from 'react'
-import { useCart } from '../../contexts/cart-context';
+import React from "react";
+import { useCart } from "../../contexts/cart-context";
 
-const CartSummary=()=> {
-    const {cartState:{cart}}=useCart();
-    const totalAmt=cart.reduce((acc,cur)=> Number(cur.price) * Number(cur.qty) + acc,0);
-    const totalItem=cart.reduce((acc,cur)=> Number(cur.qty) + acc,0);
-    const deliveryCharges=100;
-    const finalAmt=totalAmt+deliveryCharges;
+const CartSummary = () => {
+  const {
+    cartState: { cart },
+  } = useCart();
+  const totalAmt = cart.reduce(
+    (acc, cur) => Number(cur.price) * Number(cur.qty) + acc,
+    0
+  );
+  const totalItem = cart.reduce((acc, cur) => Number(cur.qty) + acc, 0);
+  const deliveryCharges = 100;
+  const finalAmt = totalAmt + deliveryCharges;
 
   return (
     <div class="billing-details">
-    <h2>PRICE DETAILS</h2>
-    <hr />
-    <div class="v-space-1rem"></div>
-    <div class="flex-row space-between">
+      <h2>PRICE DETAILS</h2>
+      <hr />
+      <div class="v-space-1rem"></div>

```

Comment:
Here Class should be changed to className

Reply:
Thanks for pointing it out. 
Done [f9ab2b5](https://github.com/Manpreet-hub/ecommerce-react/pull/6/commits/f9ab2b5bb2e0009c6ae0186c252605279916d865)